packages/playback-core/package.json

```diff
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "hls.js": "1.2.3",
-    "mux-embed": "^4.7.0"
+    "mux-embed": "^4.14.0"
```

Comment:
NOTE: A version bump was necessary to get a version of mux-embed with an exposed `generateUUID` in its public `utils`. Upgrading to latest version, bc why not.